x-pack/plugin/sql/qa/src/main/resources/command.csv-spec

```diff
@@ -25,6 +25,7 @@ STDDEV_POP       |AGGREGATE
 SUM_OF_SQUARES   |AGGREGATE      
 VAR_POP          |AGGREGATE      
 HISTOGRAM        |GROUPING        
+CASE             |CONDITIONAL
```

Comment:
Do you agree with listing CASE as a conditional function?
Case is quite special and doesn't follow at all the traditional functional format.